pkg/querier/queryrange/queryrangebase/roundtrip.go

```diff
@@ -299,8 +297,8 @@ func (q roundTripper) Do(ctx context.Context, r Request) (Response, error) {
 		return nil, err
 	}
 	defer func() {
-		io.Copy(ioutil.Discard, io.LimitReader(response.Body, 1024))
-		_ = response.Body.Close()
+		io.Copy(ioutil.Discard, io.LimitReader(response.Body, 1024)) //nolint:errcheck
```

Comment:
how do you feel about moving the exclusion to the `.golangci.yml`? No big preference on my part, but it is a little nicer to avoid the comments I guess.

Reply:
Hmmm I checked our `.golanci.yaml` and there are file exclusions there, so the option would be to exclude the whole error message regarding `io.Copy` but I'm not sure if we want that, as it might be useful in other contexts. That said, I think it makes sense to keep the exclusion as a comment. WDYT?

Reply:
doesn't `_ = ....` solves it ?

Reply:
`_ = ...` worked great, ty!